WordPress/WordPressStatsWidgets/Views/Localization/LocalizableStrings.swift

```diff
@@ -57,6 +57,31 @@ enum LocalizableStrings {
                                                     value: "Unable to load site stats.",
                                                     comment: "Title of the nodata view in today widget")
 
+    static let noDataViewTodayTitle = AppLocalizedString("widget.today.nodata.view.title",
+                                                         value: "Unable to load today's stats.",
+                                                         comment: "Title of the nodata view in today widget")
```

Comment:
nitpick: I think changing this to "...the 'no data' view..." will provide more clarity to translators.